components/eamxx/src/share/grid/mesh_free_grids_manager.cpp

```diff
@@ -2,6 +2,9 @@
 #include "share/grid/point_grid.hpp"
 #include "share/grid/se_grid.hpp"
 #include "share/grid/remap/do_nothing_remapper.hpp"
+#include "share/io/scorpio_input.hpp"
+
+#include "physics/share/physics_constants.hpp"
```

Comment:
I am reluctant to include physics/share stuff in the scream_share library. This is a header only, so no circular deps, but it's still confusing.

I see that you only need it for pi. Perhaps pi could be moved in `share/utils/scream_universal_constants.hpp`? It's fine to store it as a double, and let the assignment operator cast it down to float when doing `Real pi = constants::pi;`.

Reply:
Note: for bwd compatibility, we can put
```
static constexpr Scalar Pi = constants::pi;
```
in `physics/share/physics_constants.hpp`, so that we don't have to change all the files using pi.